utils/python/tests/test.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+import sys, os
+cimeroot = os.getenv("CIMEROOT",\
+        os.path.dirname(os.path.abspath(os.path.join(__file__,"..",".."))))
```

Comment:
Why not add this to scripts_regression_tests so it can be run with everything else?